app/ui/browser/views/navbar/navbar.jsx

```diff
@@ -207,7 +208,7 @@ NavBar.displayName = 'NavBar';
 
 NavBar.propTypes = {
   page: PropTypes.object,
-  pages: PropTypes.object.isRequired,
+  numPages: PropTypes.number.isRequired,
```

Comment:
Maybe `pageCount` or something? `numPages` isn't clear what it was representing to me